src/test/rustdoc-gui/label-next-to-symbol.goml

```diff
@@ -0,0 +1,26 @@
+goto: file://|DOC_PATH|/test_docs/index.html
```

Comment:
Please add a comment explaining what this test is doing overall.

Reply:
I added two general lines. Should they start with `///` or `//!` or `//`?